### DISCUSSION TITLE
How to handle variants in Pimcore

### DISCUSSION THREAD

**Discussion by KasthuriAnandhan** [pimcore/pimcore#16146] [Q&A / Support]
@kingjia90 & @dvesh3 

hello Guys,

Can you please provide how to create variants and how to handle that in both pimcore and PXP, I have referred the pimcore documentation and followed the instructions.

Enabled the below things 
![Screenshot 2023-10-25 152452](https://github.com/pimcore/pimcore/assets/125338676/bcef64dd-3919-4390-a23e-51168f28d0d1)

and created variants for the below product.
![Screenshot 2023-10-25 152717](https://github.com/pimcore/pimcore/assets/125338676/0d1397bd-e3c5-4900-8ebf-7daa3ca90c89)

The variants are showedcin the products page like below, is this the way to show variants in PXP page.
![Screenshot 2023-10-25 153017](https://github.com/pimcore/pimcore/assets/125338676/32078e47-26db-45c1-a736-67375136de15)

There is any other option are there to select variants from it's parent product because it shows like 3 different products.

**Comment by KasthuriAnandhan** [comment]
Hello anybody can help here

**Reply by kingjia90** [reply]
Sorry, i didn't get it, you want in PXP to group the variants to be hidden under the main one and that it can  somehow expand/collapse on click, right?
If so, i am not aware of this feature yet.

**Comment by KasthuriAnandhan** [comment]
Hi @kingjia90 & @dvesh3

When I import variant product under parent product through datahub object importer that was placed in under children grid of the data object does not there under in variants but I need to there under variants tab of the data object.

![image](https://github.com/pimcore/pimcore/assets/125338676/9b238ab4-ead7-49c7-9fbc-f96e64de581e)

Under variants tab no product is there how to change it
![image](https://github.com/pimcore/pimcore/assets/125338676/77a7203a-fd71-4b15-a6a1-19436bd1f6e1)
